contracts/contracts/Battle.sol

```diff
@@ -187,6 +190,23 @@ contract Battle {
 
   }
 
+  /// @dev Mints an NFT based on the winner
+  /// @dev Interface/Delegate Call out To IGameItems
+  /// @dev Contract Becomes Signer with WIN_MANAGER_ROLE
+  /// @param itemContract Game Items Contract
+  /// @param result of the battle, 0 = P1 Win, 1 = P2 Win, 2 = Skipped (Draw)
+  /// @param p1TokenId Player One Token Id
+  /// @param p1 Player One Address
+  /// @param p2 TokenId Player Two TokenId
+  /// @param p2 Player Two Address
+  function _mintForWinner(IGameItems itemContract, uint8 result, uint256 p1TokenId, address p1, uint256 p2TokenId, address p2) internal {
+    if (result == 0) {
```

Comment:
what happens at draws?

Reply:
> Currently we have no logic for a draw. If we want we can build an extra card that is provided on draws specifically for ties to have an "action"

Hmm not sure I understand and I'll have to look, but we definitely have draws (like both players play rock).

They just shouldn't get any card I think for now and in the future maybe we can add more fun logic there.